### PR TITLE
First backbone

### PR DESCRIPTION
**Pull request by Usagib**
code review pull request for project: Displaying and inputting data

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+<head>
+	<meta charset="utf-8">
+	<title>awesome-formulary</title>
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+<body>
+	<div class="nav-bar">
+		<div>
+			<button class="sign-button">
+			<!--	<img class="s-logo" src="https://cdn4.iconfinder.com/data/icons/epic-outlines/30/667814-lock_4-512.png" alt="" /> -->
```
Comment (samgaco) [inline index.html:13]: It is better to avoid commented code in your projects.

**Review by samgaco — CHANGES_REQUESTED** [review]
### Hi @Usagib and @sebastianlujan :hand: 

### Great work :clap: and well-done readme file :+1: ,  only a few points to address before proceeding:

* Correct the errors in the index file given to you by the [validator](https://validator.w3.org).
* In your [project](https://rawcdn.githack.com/Usagib/DisplayingandInputtingData/ff989e0c43f8074aa51984ac6055437ee3627a6f/index.html) the bottom part of the white box doesn't look the same as in the original:

> intuit

* Fix the linters checks that have failed

* **[SUGGESTED]** You could add a live demo of the site in your readme file using https://raw.githack.com/ for example

### Please :pray: submit back for another code review when you are done with these changes :muscle:

**Comment by Usagib** [comment]
Hello Team!
we made the proper changes:
- We passed the validator test
- Corrected lint style errors
- Added some margins and background color to the form footer
- Added a LiveDemo using GitHub Pages!

Heading out the pull request for code review, Happy coding!

**Review by Oghenebrume50 — APPROVED** [review]
Hi team this is approved, good Job :+1:

**Pull request merged**